.github/workflows/pr.yaml

```
@@ -92,6 +92,34 @@ jobs:
         run: |
           composer coding-standards-check
 
+  coding-standards-twig-cs-fixer:
+    name: Twig - Check Coding Standards
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Setup Twig, with composer and extensions
+        uses: vincentlanglet/twig-cs-fixer
+        with:
+          coverage: none
+          tools: composer:v2
```

This block should set up PHP (`composer install` below takes care of  installing `vincentlanglet/twig-cs-fixer`):

```suggestion
      - name: Setup PHP, with composer and extensions
        uses: shivammathur/setup-php@v2
        with:
          php-version: 8.2
          extensions: ctype, dom, iconv, json, zip, gd, soap
          coverage: none
          tools: composer:v2
```